src/porepy/models/constitutive_laws.py

```diff
@@ -2181,17 +2183,35 @@ def well_flux_equation(self, interfaces: list[pp.MortarGrid]) -> pp.ad.Operator:
         r_e = self.equivalent_well_radius(subdomains)
 
         f_log = pp.ad.Function(pp.ad.functions.log, "log_function_Piecmann")
-        e_i = self.e_i(subdomains, i=0, dim=9).T  # type: ignore[call-arg]
+
         # We assume isotropic permeability and extract xx component.
+        e_i = self.e_i(subdomains, i=0, dim=9).T  # type: ignore[call-arg]
+
+        # To get a transmissivity, we multiply the permeability with the length of the
+        # well within one cell. For a 0d-2d coupling, this will be the aperture of the
+        # 2d fracture cell; in practice the number can be obtained by multiplying with
+        # the specific volume, or by taking a volume integral over the mortar cell
+        # (which will incorporate the specific volume of the higher-dimensional
+        # neighbor, that is, the fracture). For a 1d-3d coupling, we will need the
+        # length of the well within the 3d cell (see the MRST book, p.128, for comments
+        # regarding deviated wells). Again, this could be obtained by a volume integral
```

Comment:
Since both cases could be covered by volume integral, I don't see why you change from that to multiplication by specific volume.

Reply:
I'll change to the specific volume of the interface. To me this is more explicit (though of course equivalent) to the volume integral.

Reply:
Now I got it. It will be a volume integral.